ext/gmp/gmp.c

```diff
@@ -910,8 +927,8 @@ int gmp_import_export_validate(zend_long size, zend_long options, int *order, in
 			*endian = 0;
 			break;
 		default:
-			php_error_docref(NULL, E_WARNING,
-				"Invalid options: Conflicting word endianness");
+			/* options argument is in second position */
+			zend_argument_value_error(3, "has conflicting word endianness");
```

Comment:
_maybe_ someting like `GMP_BIG_ENDIAN and GMP_LITTLE_ENDIAN cannot be defined in the same time`. These formats don't really support the format of `zend_argument_value_error()` though.

Reply:
I went with another variant which can be used more often